packages/server/src/api/controllers/deploy/aws.js

```diff
@@ -22,11 +22,42 @@ async function invalidateCDN(cfDistribution, appId) {
     .promise()
 }
 
-exports.fetchTemporaryCredentials = async function() {
+exports.updateDeploymentQuota = async function(quota) {
+  const DEPLOYMENT_SUCCESS_URL =
+    process.env.DEPLOYMENT_CREDENTIALS_URL + "deploy/success"
```

Comment:
I've been trying to pass all new environment variables through the `environment.js` file, purely again so its all centralised, I think if we manage it that way going forward it'll be much easier for self hosting to work out what is needed.